### DISCUSSION TITLE
Doubt regarding Copilot

### DISCUSSION THREAD

**Discussion by Greatz08** [teableio/teable#1058] [General]
![image](https://github.com/user-attachments/assets/32c6a50e-ce90-4099-908f-9908e7820b76)
Is copilot thing related to open AI or microsoft copilot or is it related to open source llm integration for doing those mentioned specific task ?

**Comment by tea-artist** [comment]
I haven't started developing these features yet. The expectation is that there will be models for users to choose from.
